repo/ssh.py

```diff
@@ -52,6 +52,10 @@ def version():
   """return ssh version as a tuple"""
   try:
     return _parse_ssh_version()
+  except FileNotFoundError:
+    print('warning: no ssh installed')
+    # don't crash if ssh is not present in the system, and the manifest is not using ssh urls
+    return (8,2)  # assume recent version
   except subprocess.CalledProcessError:
```

Comment:
I think you also should make sure that this exception does not stop repo. I presume it depends on the OS